uiowa/uiowa
# now.uiowa.edu Article Type site split

Relates to the first slice of #5474 





# How to test


`git checkout now-news && git pull`
`blt ds --site=now.uiowa.edu`
`drush @now.local en now_core smart_date`
`drush @now.local cim --source ../config/sites/now.uiowa.edu --partial`
`drush @now.local cim`
`drush @now.local cr`

Try adding some new articles with the various article types. Article Type should be required, defaulting to News Feature.

- `In the News` adds an "Original Publication Date" field
- `Achievement` currently matches the default Article type
- `News Feature` adds an "Embargo Message" field
  - If there is content in the Embargo Message field, the news feature should not be able to be published.
